src/Configuration/Programmer/Programmer.cpp

```diff
@@ -221,6 +227,49 @@ void programmer_entry(const CFGCommon_ARG* cmdarg) {
         CFG_POST_ERR("Failed to program FPGA. Error code: %d", status);
         return;
       }
+    } else if (subCmd == "otp") {
+      auto otp_arg =
+          static_cast<const CFGArg_PROGRAMMER_OTP*>(arg->get_sub_arg());
+      if (otp_arg->confirm == false) {
+        std::string prompt;
+        std::cout << "This OTP programming is not reversable." << std::endl
+                  << "Do you want to continue? [Y/n]";
+        std::cin >> prompt;
```

Comment:
n should be capital, consistent with Y

Reply:
@alain-rs is this the right way to get user input for tcl commands ?

Reply:
This user prompting to proceed mechanism is for the interactive sessions (e.g., batch mode & FOEDAG/Raptor UI mode) where user interaction is expected. For the concern of automation (e.g., scripting mode), the command provides a -y flag to gather consensus to proceed and the prompting will be bypassed. (similar UX like Ubuntu apt install -y option).

FYI, this is one of the user requirements from FW team as OTP programming is irreversible and we need to handle this with care for the end-user or designer.

Reply:
Thanks for explaining that, please update the help to reflect these changes and flag.

Reply:
@ravic-rs this is not working in the GUI mode

Reply:
Programmer GUI will use -y option to bypass the prompting. I will update the Programmer GUI spec to support this feature. Stay tuned.

Reply:
The user prompting is removed. Users must use -y flag with the OTP command to give consensus to proceed with the OTP programming. Without the -y flag, the command will stop and return immediately. Thus, with this way the command will not create unnecessary blocking if user forget the to use -y flag in scripting mode.